docs/03-domain-configuration.md

```diff
@@ -43,14 +57,20 @@ gcloud compute instances describe sigstore-fulcio \
 
 ### oauth2.example.com
 
+```bash
+$ gcloud compute instances describe sigstore-oauth2 \
+  --format='get(networkInterfaces[0].accessConfigs[0].natIP)'
 ```
-gcloud compute instances describe sigstore-oauth2 \
---format='get(networkInterfaces[0].accessConfigs[0].natIP)'
+
+```bash
+$ gcloud dns record-sets create oauth2.example.com. \
```

Comment:
I would add something here saying this is required if they are using google cloud as a DNS provider.

Reply:
done